projects/plugins/beta/src/admin/plugin-manage.template.php

```diff
@@ -149,6 +149,12 @@ class="is-primary jp-form-button dops-button is-primary is-compact jptracks"
 		}
 		?>
 		<?php
+		$branch = $plugin->source_info( 'trunk', '' );
+		if ( $branch && ! is_wp_error( $branch ) ) {
+			require __DIR__ . '/branch-card.template.php';
+		}
+		?>
+		<?php
 		$branch = $plugin->source_info( 'master', '' );
```

Comment:
Again, with the "little work" we can get rid of the 'master' case here.

As it is, when we start the transition of the key from "master" to "trunk" in the manifests this would wind up showing "Bleeding Edge" twice.